anime_downloader/sites/animixplay.py

```diff
@@ -25,18 +26,43 @@ def search(cls, query):
     def _scrape_episodes(self):
```

Comment:
Most of this should really be in AniMixPlayEpisode, this is only to get the episode links

Reply:
Upon closer inspection there's no better way to do this